mlkem/poly.c

```diff
@@ -397,17 +396,17 @@ __contract__(
 MLK_INTERNAL_API
```

Comment:
@mkannwischer Can you add a comment here? e.g.
```
/* Reference: `ntt()` in the reference implementation.
 * - Iterate over `layer` instead of `len` in the outer loop */
```

Reply:
done.